apps/cowswap-frontend/src/common/hooks/useCmsAnnouncements.ts

```diff
@@ -0,0 +1,19 @@
+import { CmsAnnouncements, getAnnouncements } from '@cowprotocol/core'
+
+import ms from 'ms.macro'
+import useSWR, { SWRConfiguration } from 'swr'
+
+const ANNOUNCEMENTS_SWR_CONFIG: SWRConfiguration = {
+  refreshInterval: ms`5min`, // we do need to show this sort of ASAP
+  refreshWhenHidden: false,
+  refreshWhenOffline: false,
+  revalidateOnFocus: false,
```

Comment:
You could put `EMPTY_VALUE` as `fallbackData` parameter so you don't need to write `data || EMPTY_VALUE`.
This just a super nitpick, feel free to ignore it

Reply:
TS doesn't seem to like that. It thinks the result can be `undefined`.

![image](https://github.com/user-attachments/assets/6d392cb1-6cee-4408-8fd7-537f9bc4fbd1)
![image](https://github.com/user-attachments/assets/b2583881-d4bc-4c38-9953-911b25a50bad)